lib/assignDisabledRanges.js

```diff
@@ -53,7 +53,41 @@ module.exports = function (root, result) {
 	};
 
 	result.stylelint.disabledRanges = disabledRanges;
-	root.walkComments(checkComment);
+
+	// Work around postcss/postcss-scss#109 by merging adjacent `//` comments
+	// into a single node before passing to `checkComment`.
+
+	/** @type {PostcssComment?} */
+	let inlineEnd;
+
+	root.walkComments((/** @type {PostcssComment} */ comment) => {
```

Comment:
This annotation is necessary since the inferred type is the unmodified `Comment` type from PostCSS, and we want to use the modified type that includes the `inline` properties.